### DISCUSSION TITLE
Linux- Debian & Raspbian Bookworm RustDesk- service

### DISCUSSION THREAD

**Discussion by IT-Betreuung** [rustdesk/rustdesk#7817] [General]
Unfortunately, the RustDesk- service no longer starts in headless mode,
I first have to register a user via Remote Desktop,
then RustDesk is "active" again.

When installing on a "fresh" machine,
but also when updating I get the following warning:
„Warning: The unit file, source configuration file or drop-ins of rustdesk.service changed on disk. Run 'systemctl daemon-reload' to reload units.“

For the following assets:
rustdesk-1.2.4-x86_64.deb
rustdesk-1.2.4-armv7-sciter.deb
rustdesk-1.2.4-aarch64.deb

**Comment by MStylesMS** [comment]
Can you check to see if you are getting excessive kworker processes created when rustdesk runs on your Debian or Raspberry Pi? I just posted a bug report on this.  Was bogging my system down after running a few hours.

You can do `ps | grep ps` and note the PID number.  Wait about 10 seconds then do it again.  Unless you have something else going on the PID should only be about 2 higher.  On my machines, it is about 100 higher, leading to tens of thousands per hour.
